app/src/main/java/org/systers/mentorship/view/activities/FilterActivity.kt

```diff
@@ -67,13 +67,13 @@ class FilterActivity : BaseActivity() {
             }
         }
 
-        if (map?.get(NEED_MENTORING_KEY) == "true") {
+        if (map?.get(NEED_MENTORING_KEY) ==  getString(R.string.True)) {
             cardFilterNeedMentoring.setBackgroundResource(R.drawable.background_rounded_primary)
             tvFilterNeedMentoring.setTextColor(ResourcesCompat.getColor(
                     resources, R.color.white, null))
             needMentoring = true
         }
-        if (map?.get(AVAILABLE_TO_MENTOR_KEY) == "true") {
+        if (map?.get(AVAILABLE_TO_MENTOR_KEY) == getString(R.string.True)) {
```

Comment:
this true value could also be a constant 🤔

Reply:
> this true value could also be a constant 🤔

So "true" should not be added as a string resource but kept as a hardcoded string? All the occurrence of "true" should be changed or only in this file?